ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -140,8 +144,14 @@
   :ssl?                 - allow connections over HTTPS
   :ssl-port             - the SSL port to listen on (defaults to 443, implies
                           :ssl? is true)
-  :exclude-ciphers      - When :ssl? is true, exclude these cipher suites
-  :exclude-protocols    - When :ssl? is true, exclude these protocols
+  :exclude-ciphers             - When :ssl? is true, additionally exclude these
+                                 cipher suites
+  :exclude-ciphers-overrides   - When :ssl? is true, only exclude these cipher
+                                 suites (takes precendence over :exclude-ciphers)
+  :exclude-protocols           - When :ssl? is true, additionally exclude these
+                                 protocols
+  :exclude-protocols-overrides - When :ssl? is true, only exclude these protocols
+                                 (takes precendence over :exclude-protocols)
   :keystore             - the keystore to use for SSL connections
```

Comment:
wasn't sure whether it would be desirable to produce a noisy diff by aligning everything,  so i just aligned the relevant lines (which were edited anyway).